ament_cobra/ament_cobra/main.py

```diff
@@ -160,8 +160,30 @@ def main(argv=sys.argv[1:]):
 
     error_count = 0
 
+    # Unfortunately, the CWE ruleset is not outputting a JSON file
+    # Issue submitted here: https://github.com/nimble-code/Cobra/issues/50
+    ruleset_to_filename = {
+        'basic': '_Basic_.txt',
+        'cwe': None,  # No output JSON to work with (need #50 fixed)
+        'p10': '_P10_.txt',
+        'jpl': '_JPL_.txt',
+        'misra2012': '_Misra2012_.txt',
+        'C++/autosar': '_Autosar_.txt',
+    }
+
+    input_filename = ruleset_to_filename[args.ruleset]
+
+    # Remove old input_filename, if it exists.
```

Comment:
I don't like this, but it's needed in Cobra's current implementation. Cobra dumps a `_Autosar_.txt` file in `spaceros_ws/src/<pkg>/` directory. Is it worth asking Cobra maintainers to update this to optionally take an extra command-line parameter, so this file can instead be put in a better location (say, somewhere in `spaceros_ws/build/<pkg>/test_results/`)?

Reply:
Yes, I think that's a good idea. They've been pretty responsive so far. We could also just add the option to Cobra and submit a PR to their repo.